apps/mobile/src/screens/ProfileScreen/ProfileScreen.tsx

```diff
@@ -71,11 +70,10 @@ function ProfileScreenInner() {
     wrapperQuery
   );
 
-  const { top } = useSafeAreaPadding();
   const { isRefreshing, handleRefresh } = useRefreshHandle(refetch);
 
   return (
-    <View style={{ flex: 1, paddingTop: top }}>
+    <View style={{ flex: 1 }}>
```

Comment:
looks like a slight regression in padding on the profile page (not sure if this is the correct file but issue is on the profile setting screen)

its a bit hard to tell from the screenshots, but in the Before, the top padding fills the safe area so it looks correct on device. in the After, the top padding is still there but no longer needed for safe space since the banner takes up the safe space

before
![IMG_2614](https://github.com/gallery-so/gallery/assets/80802871/7d83ee1b-5043-4bfc-92f1-2779eae2238f)


after
![CleanShot 2023-09-11 at 15 05 19@2x](https://github.com/gallery-so/gallery/assets/80802871/fdc925e5-6921-4277-8501-e072944be9f3)

Reply:
Nice catch @kaitoo1! Fixed the regression

<img width="442" alt="CleanShot 2023-09-11 at 15 55 37@2x" src="https://github.com/gallery-so/gallery/assets/4480258/85e9ed50-54b5-4e5c-b58f-f58e65ee4a63">